tests/security/cc/cc_audit_test_setup.pm

```diff
@@ -23,6 +23,9 @@ sub run {
 
     select_console 'root-console';
 
+    zypper_call('in SUSEConnect');
+    record_soft_failure('bsc#1193782', 'SUSEConnect is not installed when system role is common criteria');
```

Comment:
I think if `which SUSEConnect2` failed we can do `record_soft_failure` in case if `bsc#1193782` is fix and test results still show unsuitable softfailure.

Reply:
good idea